apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -419,6 +419,220 @@ public partial class BigtableClient
         }
 
         /// <summary>
+        /// Modifies a row atomically. The method reads the latest existing version
+        /// and value from the specified columns and writes a new entry based on
+        /// pre-defined read/modify/write rules. The new value for the version is the
```

Comment:
"pre-defined" => "the specified"? (Otherwise it sounds like the user doesn't get to say what happens...)

Reply:
Again these are copies of auto-generated content. I think in this context it means "defined before the request gets sent to the server" so the user doesn't think they can read a value, then decide how to modify and write the new value based on that. If you still think it should be changed though, I don't feel too strongly about it.

Reply:
Right, I wondered whether that would be the case. Possibly the best thing to do is feed back to the Bigtable team that the proto comments could be clearer, then regenerate - and copy the updated comments when they come through. But this is fine for now :)

Reply:
Ok thanks. I'll reach out to them internally.